x/bank/proto/cosmos/bank/v2/bank.proto

```
@@ -1,7 +1,65 @@
 syntax = "proto3";
 package cosmos.bank.v2;
 
+import "gogoproto/gogo.proto";
```

buf

import "gogoproto/gogo.proto": file does not exist

(COMPILE)